.github/workflows/bi-connectors.yml

```diff
@@ -0,0 +1,63 @@
+name: Build connectors for BI tools
+
+on:
+  push:
+    paths:
+      - 'bi-connectors/PowerBIConnector/**'
+      - 'bi-connectors/TableauConnector/**'
+      - '.github/workflows/bi-connectors.yml'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Pack Tableau ODBC connector
+      id: pack-tableau-odbc
+      run: |
+        zip -r opensearch_sql_odbc.taco . -x *.taco
+      working-directory: bi-connectors/TableauConnector/opensearch_sql_odbc
+    - name: Pack Tableau JDBC connector
+      id: pack-tableau-jdbc
+      run: |
+        zip -r opensearch_sql_jdbc.taco . -x *.taco
+      working-directory: bi-connectors/TableauConnector/opensearch_sql_jdbc
+    - name: Prepare Power BI ODBC connector
+      run: |
+        cp SqlOdbcPBIConnector.pq SqlOdbcPBIConnector.m
+      working-directory: bi-connectors/PowerBIConnector
+    - name: Pack Power BI ODBC connector
+      id: pack-powerbi-odbc-os-proj
+      run: |
+        zip OpenSearchProject.mez *.png *.m *.resx *.pqm
+      working-directory: bi-connectors/PowerBIConnector
+    - name: Prepare the second Power BI ODBC connector
```

Comment:
What is the second one...?

Reply:
There are 2 connector requested by Amazon: one for `OpenSearch Project` which is being build from the repo as is; and the second one - for `Amazon OpenSearch Service`. To build the second one we need to modify few strings in the connector code.

Reply:
Can you then name them accordingly instead of "the second one"?

Reply:
Changed in 6e378705.